src/game-status/adiMalka.js

```diff
@@ -0,0 +1,28 @@
+const optionToGuess = 4;
+
+/**
+ * thr function get array of strung - colors and return random array of new options
+ * @param {[string]} colorOptions
+ * @return {[string]} colorsToGuess
+ */
+function getColorsToGuess(colorOptions){
+    return new Array(optionToGuess).fill(0)
+        .map(() => colorOptions[Math.floor(Math.random()*colorOptions.length)]);
+}
+
+/**
+ * thr function get array of strung - colors and return random array of new options
```

Comment:
change the description to align with the function. it should should explain the convertion of the array